### PR TITLE
add pranay202

### PR DESCRIPTION
**Pull request by pranay202**
create my linkfree profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1780"></a>

### CODE REVIEW

File: src/config/user.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "username": ""
+  "username": "pranay202"
```
Comment (Panquesito7) [inline src/config/user.json:2]: Please revert the changes here. This change will make LinkFree not work.
```suggestion
  "username": ""
```

File: public/data/pranay202.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        "description": "Gold badge for 10 Days of JS",
+        "url": "https://www.hackerrank.com/kharabepranay"
+      },
+      {
+        "title": "Earned Linkedin skill Badge: Javascript",
+        "date": "July 2021",
+        "icon": "linkedin",
+        "color": "yellow",
+        "description": "In the top 5% of 897.2k people who took this.",
+        "url": "https://www.linkedin.com/feed/update/urn:li:activity:6953234827391250432/"
```
Comment (Panquesito7) [inline public/data/pranay202.json:56]: I don't think this link (and the one below) will work as it's in your feed activity.
You should probably add a direct link so that anyone can check it. 🙂

File: public/data/pranay202.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        "url": "https://discord.com/channels/@me/KharabePranay#9791",
+        "icon": "discord"
+    },
+    {
+        "name": "Connect on Twitter",
+        "url": "https://twitter.com/kharabePranay23",
+        "icon": "twitter"
+    },
+    {
+        "name": "Connect on Instagram",
```
Comment (Panquesito7) [inline public/data/pranay202.json:28]: I think you meant to use `LinkedIn` here.
```suggestion
        "name": "Connect on LinkedIn",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by imlakshay08 — CHANGES_REQUESTED** [review]
You need to remove the activity's links of your LinkedIn profile, you mentioned in your milestone section. You can add the link of the webpage instead.

File: public/data/pranay202.json (hunk 3 above)
Comment (pranay202) [reply public/data/pranay202.json:28]: Thank you for guiding me. I'll surely work on this and make a PR ASAP.

File: public/data/pranay202.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Pranay Kharabe",
+  "type": "personal",
+  "bio": "I'm a full stack MERN developer and an open-source enthusiast specializing in building exceptional digital experiences. Currently focused on building responsive full stack web applications.",
+  "avatar": "https://github.com/pranay202.png",
+  "links": [
+  {
+      "name": "Connect on Linkedin",
```
Comment (Panquesito7) [inline public/data/pranay202.json:8]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Connect on LinkedIn",
```

**Review by Vyvy-vi — APPROVED** [review]
LGTM!
